src/v/cluster/partition_probe.cc

```diff
@@ -153,13 +164,13 @@ void replicated_partition_probe::setup_internal_metrics(const model::ntp& ntp) {
           labels),
       },
       {},
-      {sm::shard_label});
+      {sm::shard_label, partition_label});
```

Comment:
This will also aggregate `offset` metrics like high_watermark, end_offset, etc. I think in this case the aggregation has no point, we may either remove those metrics or consider not aggregating them

Reply:
@mmaslankaprv - I was thinking that it still makes sense in that offsets are also a "count of records" in the partition, so after aggregation they will be a count of records in the topic. E.g., end_offset - start_offset is roughly the record count (if we ignore that there may be non-Kafka records in there too).

So maybe at least one of these may make sense to keep. For debugging issues, in your experience which are the most valuable offsets? Note that we do keep `redpanda_kafka_max_offset` (a true "kafka" offset, i.e., it uses the offset translator) unaggregated on the public metrics side.